### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by zapchik**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -8,6 +8,18 @@ describe(`Function 'getHumanAge':`, () => {
...
+    expect(getHumanAge(9, 8)).toEqual([0, 0]);
+  });
+
+  it(`+9 human years for second cat and dog year`, () => {
+    expect(getHumanAge(15, 24)).toEqual([1, 2]);
+  });
 
+  it(`+4 h/y for the cat's and +5 for the dog's age after 3 years`, () => {
+    expect(getHumanAge(28, 44)).toEqual([3, 6]);
   });
```
Comment (Oleksandra-Pasenchenko) [inline src/getHumanAge.test.js:24]: The test should be like ‘should return… if …’

Comment (Oleksandra-Pasenchenko) [reply src/getHumanAge.test.js:24]: Also, there can be more edge cases, please add some

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -7,7 +7,31 @@ describe(`Function 'getHumanAge':`, () => {
...
+    expect(getHumanAge(9, 8)).toBeInstanceOf(Array);
+  });
+
+  it(`15 human years for first cat and dog year`, () => {
+    expect(getHumanAge(9, 8)).toEqual([0, 0]);
+  });
+
+  it(`15 human years for first cat and dog year`, () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
```
Comment (Oleksandra-Pasenchenko) [inline src/getHumanAge.test.js:20]: but you don't pass the value '15'?

File: src/getHumanAge.test.js [hunk 3]
```diff
@@ -7,7 +7,31 @@ describe(`Function 'getHumanAge':`, () => {
...
+    expect(getHumanAge(28, 44)).toEqual([3, 6]);
+  });
+
+  it(`Old people, old pets`, () => {
+    expect(getHumanAge(72, 74)).toEqual([14, 12]);
+  });
 
+  it(`continuity of generations`, () => {
+    expect(getHumanAge(1, 89)).toEqual([0, 15]);
   });
```
Comment (Oleksandra-Pasenchenko) [inline src/getHumanAge.test.js:36]: It's hard to understand from the test title what you are testing

**Review by Oleksandra-Pasenchenko — CHANGES_REQUESTED** [review]
Relates to all tests:
The tests should be like ‘should return… if …’

File: src/getHumanAge.test.js (hunk 2 above)
Comment (zapchik) [reply src/getHumanAge.test.js]: Fixed.
